src/components/generic/Modal/Modal.scss

```diff
@@ -1,5 +1,32 @@
-.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-modal {
-  .md-modal__content {
-    padding: 2rem;
+.modal {
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 9;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-modal-content {
+    border: var(--wxc-modal-border);
+    box-shadow: 0px 24px 48px rgba(0, 0, 0, 0.2), 0px 0px 1px rgba(0, 0, 0, 0.12);
```

Comment:
we have a color code: $wxc-black-alpha-20

but maybe the whole box shadow should be a css variable to allow customization

Reply:
and use rems